devops/dockerfiles/node/Dockerfile

```diff
@@ -3,23 +3,36 @@ LABEL description="Compiles all workspace artifacts"
 WORKDIR /mangata
 COPY . /mangata
 
+#Export memory Profile env. variables.
+RUN export MEMORY_PROFILER_LOG=info && export MEMORY_PROFILER_LOGFILE=profiling_%e_%t.log && export MEMORY_PROFILER_OUTPUT=profiling_%e_%t.dat && export MEMORY_PROFILER_CULL_TEMPORARY_ALLOCATIONS=1
```

Comment:
please built in dockerfile `ENV` command 
```
ENV MEMORY_PROFILER_LOG=info
ENV MEMORY_PROFILER_LOGFILE=profiling_%e_%t.log
...
```

Reply:
+1